content/4_Gold/DSU.mdx

```diff
@@ -172,20 +178,35 @@ class DisjointSets:
 		y_root = self.find(y)
 		if x_root == y_root:
 			return False
-		
-		if self.sizes[x_root] < self.sizes(y_root):
+
+		if self.sizes[x_root] < self.sizes[y_root]:
 			x_root, y_root = y_root, x_root
 		self.parents[y_root] = x_root
 		self.sizes[x_root] += self.sizes[y_root]
 		return True
+
+	# returns whether two nodes are in the same connected component
+	def connected(self, x: int, y: int) -> bool:
+		return self.find(x) == self.find(y)
 ```
 
 </PySection>
 </LanguageSection>
 
-## Focus Problem Solution
+## Solution - Focus Problem
 
-<FocusProblem problem="sample" />
+**Time Complexity:** $\mathcal{O}(Q \log ^*N)$
+
+Without union find, we would have to represent the graph with an adjacency list
+and use [flood fill](https://usaco.guide/silver/flood-fill) to calculate
+connected components. This approach takes $\mathcal{O}(NQ)$ time, which is too
+slow, motivating us to use **union find**.
+
+By representing the graph with the union find data structure, we can use its
+methods to both _unite_ vertices and _find_ if two vertices $u_i$ and $v_i$ are
```

Comment:
"find" probably shouldn't be italicized (its inaccurate to say that's why it's called "find")